versions/DEVELOPMENT-SNAPSHOT-2016-10-13-a.yaml

```diff
@@ -1,5 +1,3 @@
 binaries:
-  osx: https://swift.org/builds/development/xcode/swift-DEVELOPMENT-SNAPSHOT-2016-10-13-a/swift-DEVELOPMENT-SNAPSHOT-2016-10-13-a-osx.pkg
   ubuntu14.04: https://swift.org/builds/development/ubuntu1404/swift-DEVELOPMENT-SNAPSHOT-2016-10-13-a/swift-DEVELOPMENT-SNAPSHOT-2016-10-13-a-ubuntu14.04.tar.gz
   ubuntu15.10: https://swift.org/builds/development/ubuntu1510/swift-DEVELOPMENT-SNAPSHOT-2016-10-13-a/swift-DEVELOPMENT-SNAPSHOT-2016-10-13-a-ubuntu15.10.tar.gz
-  ubuntu16.04: https://swift.org/builds/development/ubuntu1604/swift-DEVELOPMENT-SNAPSHOT-2016-10-13-a/swift-DEVELOPMENT-SNAPSHOT-2016-10-13-a-ubuntu16.04.tar.gz
```

Comment:
It's weird, some of these URLs actually work but are no longer listed in the swift.org/download page.

I'm not sure what to do.